crates/sui-indexer-alt/src/pipeline/concurrent/collector.rs

```diff
@@ -82,7 +82,7 @@ impl<H: Handler> From<Indexed<H>> for Pending<H> {
 /// closed.
 pub(super) fn collector<H: Handler + 'static>(
```

Comment:
the doc comment L73-L76 seems outdatad as now codes have MIN_EAGER_ROWS MAX_CHUNK_ROWS and MAX_PENDING_ROWS